sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample07_QueryDetectedAnomalies.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.AI.MetricsAdvisor.Models;
+using Azure.AI.MetricsAdvisor.Tests;
+using Azure.Core.TestFramework;
+
+namespace Azure.AI.MetricsAdvisor.Samples
+{
+    public partial class MetricsAdvisorSamples : MetricsAdvisorTestEnvironment
+    {
+        [RecordedTest]
+        public async Task GetAnomaliesForDetectionConfigurationAsync()
+        {
+            string endpoint = MetricsAdvisorUri;
+            string subscriptionKey = MetricsAdvisorSubscriptionKey;
+            string apiKey = MetricsAdvisorApiKey;
+            var credential = new MetricsAdvisorKeyCredential(subscriptionKey, apiKey);
+
+            var client = new MetricsAdvisorClient(new Uri(endpoint), credential);
+
+            string detectionConfigurationId = DetectionConfigurationId;
+
+            var startTime = DateTimeOffset.Parse("2020-01-01T00:00:00Z");
+            var endTime = DateTimeOffset.UtcNow;
+            var options = new GetAnomaliesForDetectionConfigurationOptions(startTime, endTime)
+            {
+                TopCount = 3
+            };
+
+            int anomalyCount = 0;
+
+            await foreach (DataAnomaly anomaly in client.GetAnomaliesForDetectionConfigurationAsync(detectionConfigurationId, options))
+            {
+                Console.WriteLine($"Anomaly at timestamp: {anomaly.Timestamp}");
+                Console.WriteLine($"Severity: {anomaly.Severity}");
+                Console.WriteLine("Series key:");
+
+                foreach (KeyValuePair<string, string> keyValuePair in anomaly.SeriesKey.AsDictionary())
+                {
+                    Console.WriteLine($"  Dimension '{keyValuePair.Key}': {keyValuePair.Value}");
+                }
+
+                Console.WriteLine();
+
+                // Print at most 3 anomalies.
+                if (++anomalyCount >= 3)
+                {
+                    break;
+                }
+            }
+        }
+
+        [RecordedTest]
+        public async Task GetAnomaliesForAlertAsync()
+        {
+            string endpoint = MetricsAdvisorUri;
+            string subscriptionKey = MetricsAdvisorSubscriptionKey;
+            string apiKey = MetricsAdvisorApiKey;
+            var credential = new MetricsAdvisorKeyCredential(subscriptionKey, apiKey);
+
+            var client = new MetricsAdvisorClient(new Uri(endpoint), credential);
+
+            string alertConfigurationId = AlertConfigurationId;
+            string alertId = AlertId;
+
+            #region Snippet:GetAnomaliesForAlertAsync
+            //@@ string alertConfigurationId = "<alertConfigurationId>";
+            //@@ string alertId = "<alertId>";
+
+            var options = new GetAnomaliesForAlertOptions() { TopCount = 3 };
```

Comment:
Are we looking into changing the name of `TopCount` to something like `MaxPerPage`?

Reply:
It's not high priority right now, but it's still on my radar. It's listed in the "design considerations" issues. I still need to filter the stuff listed in there and move it to the public repo.

Regarding the `TopCount`, we need first to double check with the service to make sure if that's the actual intended behavior. If it's not, it would be a service side bug, and no rename would be needed.